lib/membrane/rtp/metrics.ex

```diff
@@ -3,34 +3,66 @@ defmodule Membrane.RTP.Metrics do
   Defines list of metrics, that can be aggregated based on events from membrane_rtp_plugin.
   """
 
+  alias Telemetry.Metrics
+
   @doc """
   Returns list of metrics, that can be aggregated based on events from membrane_rtp_plugin.
   """
-  @spec metrics() :: [Telemetry.Metrics.t()]
+  @spec metrics() :: [Metrics.t()]
   def metrics() do
     [
-      Telemetry.Metrics.counter(
+      Metrics.counter(
         "inbound-rtp.keyframe_request_sent",
         event_name: [Membrane.RTP, :rtcp, :fir, :sent]
       ),
-      Telemetry.Metrics.counter(
+      Metrics.counter(
         "inbound-rtp.packets",
         event_name: [Membrane.RTP, :packet, :arrival]
       ),
-      Telemetry.Metrics.sum(
+      Metrics.sum(
         "inbound-rtp.bytes_received",
         event_name: [Membrane.RTP, :packet, :arrival],
         measurement: :bytes
       ),
-      Telemetry.Metrics.last_value(
+      Metrics.last_value(
         "inbound-rtp.encoding",
         event_name: [Membrane.RTP, :inbound_track, :new],
         measurement: :encoding
       ),
-      Telemetry.Metrics.last_value(
+      Metrics.last_value(
         "inbound-rtp.ssrc",
         event_name: [Membrane.RTP, :inbound_track, :new],
         measurement: :ssrc
+      ),
+      Metrics.counter(
+        "inbound-rtp.nack",
+        event_name: [Membrane.RTP, :rtcp, :nack, :sent]
+      ),
+      Metrics.counter(
+        "inbound-rtcp.packets",
+        event_name: [Membrane.RTP, :rtcp, :arrival]
+      ),
+      Metrics.sum(
+        "inbound-rtcp.bytes_received",
+        event_name: [Membrane.RTP, :rtcp, :arrival],
+        measurement: :bytes
+      ),
+      Metrics.counter(
+        "outbound-rtcp.packets",
+        event_name: [Membrane.RTP, :rtcp, :sent]
+      ),
+      Metrics.sum(
+        "outbound-rtcp.bytes_sent",
+        event_name: [Membrane.RTP, :rtcp, :sent],
+        measurement: :bytes
+      ),
```

Comment:
The metrics you've added are on a receiver (inbound) end, so I would rename them, as they might be confused with RTCP sent as sender feedback (mostly in OutboundTrackingSerializer for now). I think the pattern should rather be `inbound-rtp.rtcp_sent` or `inbound.rtcp_sent`. This resembles stats in `wertc-internals` where such info is aggregated under `inbount-rtp` and `remote-inbound-rtp` (we may also follow that convention).